### PR TITLE
Fix the dashboard UI issues

### PR DESCRIPTION
**Pull request by robin-thomas**

### CODE REVIEW

**Comment by robin-thomas** [comment]
Fixes:
* https://github.com/nearprotocol/near-explorer/issues/33
* https://github.com/nearprotocol/near-explorer/issues/32

File: frontend/package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "dev": "next",
     "build": "next build",
-    "start": "next start"
+    "start": "next start",
+    "start:dev": "env WAMP_NEAR_EXPLORER_URL=wss://near-explorer-wamp.onrender.com/ws WAMP_NEAR_EXPLORER_FRONTEND_SECRET=Uitoo4Xe next"
```
Comment (frol) [inline frontend/package.json]: Well, `start:dev` is a confusing name. How do you feel about `dev:with-production-backend`? (It is the best non-confusing naming I came up with :disappointed: )

**Comment by frol** [comment]
Feel free to squash and merge the commit.

File: frontend/package.json (hunk 1 above)
Comment (robinth91) [reply frontend/package.json]: A bit longer though 😓 
Ideal to have something smaller.

**Pull request merged**